src/common/app-paths/app-paths.test.ts

```diff
@@ -56,6 +56,10 @@ describe("app-paths", () => {
     });
   });
 
+  afterEach(() => {
+    builder.quit();
+  });
+
```

Comment:
No, this implies that there is some shared global state in the builder. We need to figure out what that is and remove it. There shouldn't be any requirement for cleanups after test.